proto/plan_common.proto

```diff
@@ -64,9 +53,14 @@ message ColumnDesc {
 
   // This field is used to represent the connector-spec additional column type.
   // UNSPECIFIED or unset for normal column.
-  AdditionalColumnType additional_column_type = 9;
+
+  // deprecated, use AdditionalColumn instead
+  // AdditionalColumnType additional_column_type = 9;
+  reserved 9;
 
   ColumnDescVersion version = 10;
```

Comment:
Do we need to add a new `ColumnDescVersion`? TBH I'm not sure about why it's added, and it doesn't seem to be needed here. Ask just in case.

Reply:
the field is introduced in #13707 to deal with `DEFAULT_KEY_COLUMN_NAME` change in future. 
discussions are available here https://github.com/risingwavelabs/risingwave/pull/13707#discussion_r1429947537